daemon/tests/happy_path.rs

```diff
@@ -259,6 +259,26 @@ async fn maker_rejects_rollover_of_open_cfd() {
     assert_next_state!(order_id, maker, taker, CfdState::Open);
 }
 
+#[tokio::test]
+async fn open_cfd_is_refunded() {
+    let _guard = init_tracing();
+    let oracle_data = OliviaData::example_0();
+    let (mut maker, mut taker, order_id) =
+        start_from_open_cfd_state(oracle_data.announcement()).await;
+
+    deliver_event!(maker, taker, Event::CommitFinality(order_id));
+    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
```

Comment:
Arent' we waiting as part of `assert_next_state`?

Reply:
I was following a pattern that I noticed above. I can remove all these sleeps if they don't help.

Reply:
the sleeps are necessary in most cases, so just removing them should result in breaking at least some tests.. I'm happy for them to be inside the `assert_next_state` (much less clutter, for the price of some needless sleeps). 

Of course check first, maybe we'll get away without it! :)

Reply:
Perhaps in a follow-up if anyone is motivated :D

Reply:
I'm not sure what the best way forward is in this case, so I will just leave it as is. I think the end goal should be to not need any explicit `sleep`s in these tests, but maybe that's just not feasible here.